client/src/sections/quests/Level7.tsx

```diff
@@ -24,21 +24,21 @@ const getZoneDisplay = (
 ): JSX.Element | undefined => {
   if (evil > 0) {
     return (
-      <Line href="/crypt.php">
+      <Box key={zone} href="/crypt.php">
         <b>{zone}:</b> {evil}/50 evil. <i>{quickInfo}</i>
         <HStack>
           <Divider orientation="vertical" />
           {evil > 25 ? (
             <List>
               {zoneStrategy.map((strat) => (
-                <ListItem>{strat}</ListItem>
+                <ListItem key={strat}>{strat}</ListItem>
```

Comment:
I don't think you can use `string[]` for a key. You can do `strat.join("|")` or something.

Reply:
I think `strat` is just a string from the `zoneStrategy` array, no?

Reply:
Ah, I misread, disregard